sys/syz-extract/linux.go

```diff
@@ -80,11 +80,18 @@ func (*linux) prepareArch(arch *Arch) error {
 	}
 	kernelDir := arch.sourceDir
 	makeArgs := build.LinuxMakeArgs(arch.target, "", "", "", arch.buildDir)
-	out, err := osutil.RunCmd(time.Hour, kernelDir, "make", append(makeArgs, "defconfig")...)
-	if err != nil {
-		return fmt.Errorf("make defconfig failed: %w\n%s", err, out)
+	if arch.configFile != "" {
+		out, err := osutil.RunCmd(time.Hour, kernelDir, "cp", "-f", arch.configFile, arch.buildDir)
```

Comment:
Please don't shell for such trivial operations. Go can perfectly copy files, not any worse than cp.

Reply:
use copyFile now